### PR TITLE
Remove `use-write-core` configuration

### PR DESCRIPTION
**Pull request by welovej**
Resolves https://github.com/Azure/autorest.csharp/issues/5155

### CODE REVIEW

File: samples/AnomalyDetector/src/Generated/Models/AlignPolicy.Serialization.cs [hunk 1]
```diff
@@ -19,14 +19,22 @@ public partial class AlignPolicy : IUtf8JsonSerializable, IJsonModel<AlignPolicy
         void IUtf8JsonSerializable.Write(Utf8JsonWriter writer) => ((IJsonModel<AlignPolicy>)this).Write(writer, ModelSerializationExtensions.WireOptions);
 
         void IJsonModel<AlignPolicy>.Write(Utf8JsonWriter writer, ModelReaderWriterOptions options)
```
Comment (pshao25) [inline samples/AnomalyDetector/src/Generated/Models/AlignPolicy.Serialization.cs:21]: Remember delete all the configuration i, e.g., test\CadlRanchProjects\type\model\inheritance\enum-discriminator\tspconfig.yaml

Comment (welovej) [reply samples/AnomalyDetector/src/Generated/Models/AlignPolicy.Serialization.cs:21]: I did a global research in vscode and there is only one 'use-write-core: true' left.

**Review by live1206 — APPROVED** [review]

**Comment by live1206** [comment]
The change LGTM, let's verify the regen in azure-sdk-for-net before merging.

**Review by ArcturusZhang — APPROVED** [review]

**Pull request merged**